js/index.js

```diff
@@ -39,4 +39,106 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+const nav1 = document.getElementsByTagName("a")[1].innerHTML=`${siteContent.nav["nav-item-1"]}`;
+const nav2 = document.getElementsByTagName("a")[2].innerHTML=`${siteContent.nav["nav-item-2"]}`;
+const nav3 = document.getElementsByTagName("a")[3].innerHTML=`${siteContent.nav["nav-item-3"]}`;
+const nav4 = document.getElementsByTagName("a")[4].innerHTML=`${siteContent.nav["nav-item-4"]}`;
+const nav5 = document.getElementsByTagName("a")[5].innerHTML=`${siteContent.nav["nav-item-5"]}`;
+const nav6 = document.getElementsByTagName("a")[5].innerHTML=`${siteContent.nav["nav-item-6"]}`;
+
+
+
+
+ const newNav = document.getElementsByTagName("nav")[0]
+
+ let new1 = document.createElement("a")
+
+ new1.textContent="Hello"
+
+ newNav.appendChild(new1);
+
+ const newNav1 = document.getElementsByTagName("nav")[0]
+
+ let new2 = document.createElement("a")
+
+ new2.textContent="More"
+
+ newNav1.prepend(new2);
+
+
+
+
+ const nav1Green = document.getElementsByTagName("a")[0].style.color="green"
+const nav2Green = document.getElementsByTagName("a")[1].style.color="green"
```

Comment:
Comments could be used here to tell yourself and other devs what exactly is happening here.